File: Boca/JAVA_2/Q9/src/DVD.java
public class DVD extends Produto {
    private int duracao;

    public DVD (String nome, double preco, int duracao) {
        super(nome, preco);
        this.duracao = duracao;
    }

    public int getDuracao() {
        return duracao;
    }
    

    @Override
    public String toString() {
        return super.toString() + String.format(" - %d minutos", duracao);
    }

    @Override
    public boolean ehCaro () {
        return this.preco > 50;
    }


}

File: Boca/CPP/06/DVD.cpp
#include "DVD.hpp"

DVD::DVD(const string& nome, double preco, int duracao) : Produto(nome, preco), duracao(duracao) {}

void DVD::print(ostream &out) const {
    Produto::print(out);
    out << " - " << duracao << " minutos";
}

ostream& operator<< (ostream &out, DVD &dvd) {
    dvd.print(out);
    return out;
}

DVD::~DVD() {}